src/components/vlplotgroup/vlplotgroup.js

```diff
@@ -163,6 +163,17 @@ angular.module('vlui')
           return false;
         };
 
+        scope.select = function(chart) {
+          Logger.logInteraction(Logger.actions.SPEC_SELECT, chart.shorthand, {
+            list: scope.listTitle
+          });
+          Pills.parse(chart.vlSpec);
+          if (scope.$parent.postSelectAction) {
+            scope.$parent.postSelectAction();
+          }
+          Modals.close('bookmark-list');
```

Comment:
Add  `//HACK` comment. This is okay for normal plots because bookmark isn't open yet.